packages/pn-personafisica-webapp/src/redux/delegation/reducers.ts

```diff
@@ -67,6 +78,26 @@ const delegationsSlice = createSlice({
         (e: any) => e.mandateId !== action.payload.id
       );
     });
+    builder.addCase(setDelegatesSorting, (state, action) => {
+      state.sortDelegates = action.payload;
+      state.delegations.delegates = state.delegations.delegates.sort((a: WritableDraft<Delegation>, b: WritableDraft<Delegation>) => {
```

Comment:
@AndriBoca sarebbe meglio spostare in una function l'algoritmo di sorting e richiamare la function nel builder. Il codice per Delegates e Delegators mi sembra identico

Reply:
la funzione di ordinamento è stata spostata in delegation.utility e scomposta, aggiungendo la gestione dell'ordinamento per data